paasta_tools/contrib/ide_helper.py

```diff
@@ -156,6 +269,40 @@ def install_vscode_support() -> None:
         ]
     }
     merge_vscode_settings("extensions.json", recommended_extensions)
+
+    playground_task = {
```

Comment:
just out of curiosity as a non-vscode user: what's the difference between this task thing and the other stuff above for running paasta stuff (e.g., the dict that has the `Generate deployments.json in playground` entry in it)?

Reply:
The task will allow you to run right away the whole paasta playground if the user just cares about running it, however it won't allow us to debug it.  The configurations in launch.json including ``Generate deployments.json in playground`` will allow the user to debug. For example, lets say I want to debug the api and the client, then I will run the configuration for ``paasta API playground`` and then run the configuration for ``paasta status in playground``.  The tasks.json and launch.json are both VS Code features. This is why we have the make targets for running the API playground and generating the etc_paasta_playground ( and soa_config_playground).